__device-tests__/helpers/utils.js

```diff
@@ -183,6 +183,54 @@ const clickBeginningOfElement = async ( driver: wd.PromiseChainWebdriver, elemen
 	await action.perform();
 };
 
+// long press to activate context menu
+const longPressMiddleOfElement = async ( driver: wd.PromiseChainWebdriver, element: wd.PromiseChainWebdriver.Element ) => {
+	const location = await element.getLocation();
+	const size = await element.getSize();
+
+	const action = await new wd.TouchAction( driver );
+	const x = location.x + ( size.width / 2 );
+	const y = location.y + ( size.height / 2 );
+	action.press( { x, y } );
+	action.wait( 2000 );
+	action.release();
+	await action.perform();
+};
+
+// press "Select All" in floating context menu
+const tapSelectAllAboveElement = async ( driver: wd.PromiseChainWebdriver, element: wd.PromiseChainWebdriver.Element ) => {
+	const location = await element.getLocation();
+	const action = await new wd.TouchAction( driver );
+	const x = location.x + 300;
+	const y = location.y - 50;
+	action.press( { x, y } );
+	//action.wait( 1000 );
+	action.release();
+	await action.perform();
+};
+
+// press "Copy" in floating context menu
+const tapCopyAboveElement = async ( driver: wd.PromiseChainWebdriver, element: wd.PromiseChainWebdriver.Element ) => {
+	const location = await element.getLocation();
+	const action = await new wd.TouchAction( driver );
+	const x = location.x + 220;
+	const y = location.y - 50;
+	action.press( { x, y } );
+	//action.wait( 1000 );
```

Comment:
We can probably remove this right?

Reply:
Yes, I'll push a commit to remove those. Initially, I wasn't sure if they were required, as I'd seen similar waits on examples online.

Reply:
Ok, I've removed them.